### PR TITLE
Money input: Added option to hide currencies

### PR DESCRIPTION
**Pull request by AndersGM**

### CODE REVIEW

**Comment by kaspernj** [comment]
@AndersGM Shouldn't there be a spec to test that this actually works? :-)

**Comment by AndersGM** [comment]
@kaspernj c069db6

File: npm-api-maker-inputs/__tests__/money-test.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import React from "react"
+
+Enzyme.configure({adapter: new Adapter()})
+
+test("shows currency options by default", () => {
+  const wrapper = shallow(
+    <Money currenciesCollection={["DKK"]} />
+  )
+
+  expect(wrapper.find(".component-bootstrap-money-input")).toBeTruthy();
```
Comment (kaspernj) [inline npm-api-maker-inputs/__tests__/money-test.jsx:13]:
```suggestion
  expect(wrapper.find(".component-bootstrap-money-input")).toBeTruthy()
```

File: npm-api-maker-inputs/__tests__/money-test.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+})
+
+test("hides currency options", () => {
+  const wrapper = shallow(
+    <Money currenciesCollection={["DKK"]} showCurrencyOptions={false} />
+  )
+
+  expect(wrapper.find(".component-bootstrap-money-input").exists()).toBeFalsy();
```
Comment (kaspernj) [inline npm-api-maker-inputs/__tests__/money-test.jsx:21]:
```suggestion
  expect(wrapper.find(".component-bootstrap-money-input").exists()).toBeFalsy()
```

**Pull request merged**